core/src/avm2/globals/avmplus.rs

```diff
@@ -0,0 +1 @@
+pub use crate::avm2::globals::flash::utils::get_qualified_class_name;
```

Comment:
I'd consider just implementing it in AS (by calling the flash.utils one) instead of in Rust, but this is cool too :D